portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,30 +28,37 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
-  private ArrayList<String> comments = new ArrayList<String>();
+  private List<String> comments = new ArrayList<String>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
+
     response.getWriter().println(comments);
   }
 
-
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get the input from the form.
     String text = getParameter(request, "text-input", "");
+    long timestamp = System.currentTimeMillis();
+
     comments.add(text);
 
+    Entity taskEntity = new Entity("Entry");
+    taskEntity.setProperty("Text", text);
+    taskEntity.setProperty("timestamp", timestamp);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(taskEntity);
+
```

Comment:
Move the above block for writing to datastore to a separate method.

private void persistComments(String text, long timestamp) {...}

Reply:
Done!